drivers/hmis/app/graphql/types/hmis_schema/household.rb

```diff
@@ -46,5 +49,16 @@ def enrollments
     def assessments(**args)
       resolve_assessments(**args)
     end
+
+    def staff_assignments(is_currently_assigned: true)
```

Comment:
I haven't validated any permissions here. Is it safe to assume that anyone who has view permission on the household is fine to view the caseload info about that household, or should this be more restricted?

Reply:
discussed, fine to leave as is

Reply:
add pagination (assume users will add more data than we expect)